www/src/components/Sponsors.astro

```diff
@@ -0,0 +1,40 @@
+<style lang="scss">
+.wrapper {
+  margin-bottom: 3rem;
+  margin-top: 3rem;
+}
+
+.list {
+  align-items: center;
+  display: flex;
+  justify-content: center;
+
+  a {
+    &::before,
+    &::after {
+      display: none;
+    }
+  }
+}
+
+.title {
+  font-size: 0.75em;
+  font-weight: 700;
+  text-align: center;
+  margin-bottom: 1.5rem;
+}
+
+.oc {
+  margin-top: 1.5rem;
+  font-size: 0.75em;
+  text-align: center;
+}
+</style>
+
+<div class="wrapper">
+  <h2 class="title">Sponsored by</h2>
```

Comment:
it's odd to have a header be smaller than the body text size in the copy above. can this at least match that body text size?